hs_core/views/resource_folder_hierarchy.py

```diff
@@ -71,6 +69,14 @@ def data_store_structure(request):
     res_coll = os.path.join(resource.root_path, store_path)
     try:
         store = istorage.listdir(res_coll)
+        dirs = []
+        for dname in store[0]:  # directories
+            d_pk = dname.decode('utf-8')
+            name_with_full_path = os.path.join(res_coll, d_pk)
+            istorage = resource.get_irods_storage()
+            d_url = istorage.url(os.path.join('zips', name_with_full_path))
```

Comment:
@sblack-usu Shouldn't the path for url be ```os.path.join(name_with_full_path, 'zips')``` rather than putting `zips` as the first parameter?

Reply:
no, here is the url mapping.
https://github.com/hydroshare/django_irods/blob/25c9e9cfa9912fcb88dbe7cc1b0096db28462f51/urls.py#L6

Reply:
OK. Got it. The url mapping then looks good to me.